### DISCUSSION TITLE
Migrating 8 > 13: encoding issue with p:autoComplete

### DISCUSSION THREAD

**Discussion by luccotta** [primefaces/community#1701] [PrimeFaces] [answered]
Hello,

after migrating from 8 to 13.0.8, the @FacesConverter I use for my p:autoComplete is receiving a value with encoding issues.

In getAsString I return "Português" but getAsObject is receiving "PortuguÃªs" when I select it in the p:autoComplete

Any idea why?

I'm using Tomcat 8.5, Java 11, Chrome

Thanks

**Comment by luccotta** [comment]
I found out why this is happening, my form has `enctype="multipart/form-data"`. In PF 8, it generated requests with `Content-Type: application/x-www-form-urlencoded; charset=UTF-8`.

But from PF 11 onwards, the requests are `Content-Type: multipart/form-data; boundary=----WebKitFormBoundaryAQOxGJbpLVny0u0T`

Removing the enctype solved the issue. 

But, in case I do need this `enctype` in my form, is there any workaround for this?

Thanks

**Reply by melloware** [reply]
Did you upgrade anything else like your Mojarra/MyFaces or Servlet version etc?  I don't think PrimeFaces controls that `Content-Type` .

**Reply by luccotta** [reply]
Hi, I did not. Attached is a sample project containing only PF 13.0.8 and glassfish faces 2.2.19 which my project has always used. I ran it with Tomcat 8.5. If you put a breakpoint in `getAsObject` from `LanguageConverter.java`, you can see the value arriving with encoding issues.

But when I change PF in pom.xml to 8.0 the issue doesn't happen.

Thanks
[primefaces-test.zip](https://github.com/primefaces/community/files/15065858/primefaces-test.zip)

**Reply by melloware** [reply]
It looks like this is when it changed: https://github.com/primefaces/primefaces/issues/1355

Issue: https://github.com/primefaces/primefaces/issues/6360

Issue: https://github.com/primefaces/primefaces/issues/6361

**Reply by melloware** [reply, marked as answer]
I just updated your example its working fine for me: 
[discussion-1701.zip](https://github.com/primefaces/community/files/15066311/discussion-1701.zip)

Run `mvn clean jetty:run` and navigate to http://localhost:8080/

![image](https://github.com/primefaces/community/assets/4399574/d789359c-db24-47a6-a578-8a61fa5dc6ba)

**Reply by luccotta** [reply]
Ok, thanks, I will check how you updated it.
